### DISCUSSION TITLE
Force disable Dependency Dashboard

### DISCUSSION THREAD

**Discussion by z0rc** [renovatebot/renovate#32547] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab, latest Renovate from https://gitlab.com/renovate-bot/renovate-runner.

### Please tell us more about your question or problem

I'm running Gitlab that has Issues disabled on server level. I'm looking a way to configure renovate to disable Dependency Dashboard for every repo once and for all.

Self-hosted renovate configuration has `dependencyDashboard: false`. But repo level configuration can override this by having `"extends": ["config:recommended"]`. Unfortunately this `extends` section was part of onboarding config for some time, so there are undefined number of repositories that have it `renovate.json` now.

I'm looking for a way to enforce disabled dependency dashboard on renovate side, so it cannot be enabled on repo level. As changing every "incorrect" repo configuration would be extremely tedious process.

### Logs (if relevant)

<details><summary>Logs</summary>

```

 WARN: Could not ensure issue (repository=backend/el_auto_scheduler)
       "err": {
         "name": "HTTPError",
         "code": "ERR_NON_2XX_3XX_RESPONSE",
         "timings": {
           "start": 1731602050540,
           "socket": 1731602050541,
           "upload": 1731602050541,
           "response": 1731602050595,
           "end": 1731602050595,
           "phases": {
             "wait": 1,
             "request": null,
             "firstByte": 54,
             "download": 0,
             "total": 55
           }
         },
         "message": "Response code 403 (Forbidden)",
         "stack": "HTTPError: Response code 403 (Forbidden)\n    at Request.<anonymous> (/usr/local/renovate/node_modules/.pnpm/got@11.8.6/node_modules/got/dist/source/as-promise/index.js:118:42)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)",
         "options": {
           "headers": {
             "user-agent": "RenovateBot/39.15.1 (https://github.com/renovatebot/renovate)",
             "accept": "application/json",
             "private-token": "**redacted**",
             "content-type": "application/json",
             "content-length": "4069",
             "accept-encoding": "gzip, deflate, br"
           },
           "url": "https://gitlab.example.com/api/v4/projects/backend%2Fel_auto_scheduler/issues",
           "hostType": "gitlab",
           "username": "",
           "password": "",
           "method": "POST",
           "http2": true
         },
         "response": {
           "statusCode": 403,
           "statusMessage": "Forbidden",
           "body": {"message": "403 Forbidden"},
           "headers": {
             ":status": 403,
             "cache-control": "no-cache",
             "content-type": "application/json",
             "date": "Thu, 14 Nov 2024 16:34:10 GMT",
             "strict-transport-security": "max-age=31536000; includeSubDomains",
             "vary": "Accept-Encoding, Origin",
             "x-content-type-options": "nosniff",
             "x-frame-options": "SAMEORIGIN",
             "x-gitlab-meta": "{\"correlation_id\":\"01JCNRCVFFA618E7255EWT6PAZ\",\"version\":\"1\"}",
             "x-request-id": "01JCNRCVFFA618E7255EWT6PAZ",
             "x-runtime": "0.048101",
             "content-length": "27"
           },
           "httpVersion": "2.0",
           "retryCount": 0
         }
       }

```

</details>

**Comment by rarkins** [comment]
I'm going to assume you have no dashboards created. What is the problem you're trying to solve? Eg renovate tries creating them each time? Or it skips creating them but logs a warning?

**Reply by z0rc** [reply]
Fair question. I'm trying to reduce noise in renovate logs, which as of right now filled with `WARN: Could not ensure issue` for multiple repos.

**Reply by rarkins** [reply, marked as answer]
You can use logLevelRemap centrally to downgrade that to info, debug, or trace

**Reply by z0rc** [reply]
Oh, that's new for me. Thanks, I'll check it out.
